webserver/build.gradle

```diff
@@ -11,6 +11,8 @@ dependencies {
     implementation "io.micronaut:micronaut-management"
     implementation "io.micronaut:micronaut-http-client"
     implementation "io.micronaut:micronaut-http-server-netty"
+    implementation "io.micronaut.cache:micronaut-cache-core"
```

Comment:
where the cache is configured? 
Seems that there is missing configuration on `application.yml`